test/appium/views/chat_view.py

```diff
@@ -1039,4 +1039,4 @@ def pn_invited_to_group_chat(admin, chat_name):
 
     @staticmethod
     def pn_wants_you_to_join_to_group_chat(admin, chat_name):
-        return '%s wants you to join group %s' % (admin, chat_name)
+        return '%s added you to group %s' % (admin, chat_name)
```

Comment:
Hi @ibrkhalil, Thank you very much for the PR. 

This change will only fix e2e, not the actual issue.
To fix the issue, you need to change the status go [code](https://github.com/status-im/status-go/blob/7ad0057003d968423b7ee482502cbf0e0b6716d4/protocol/local_notifications.go#L136). 
And not sure if this is in the scope of the original issue but it would also be great if the code is reformatted and cleaned up a little, as we [removed](https://github.com/status-im/status-react/pull/13428) explicit group invitation, those function names can be a little confusing (ex. `toPrivateGroupInviteNotification`)

Reply:
Understood